### PR TITLE
[DOCS] Adds ES|QL and Pandas section to the Integrations section

### PR DESCRIPTION
**Pull request by szabosteve**
## Overview

This PR adds an `ES|QL and Pandas` section to the Integrations page. The content is based on [this blog](https://www.elastic.co/search-labs/blog/esql-pandas-dataframes-python) published on Search Labs. The examples remain the same, the style of the explanatory text is adjusted slightly to be in line with the documentation style (less conversational).

I also added the index mapping request and the bulk ingest request behind closed collapsible sections, so users don't need to leave the page. @pquentin @marciw if you think it's better to exclude the requests, let me know. We can also add the content as a subpage of the Integrations page if we think it's better.

### Preview

[Integrations](https://elasticsearch-py_bk_2585.docs-preview.app.elstc.co/guide/en/elasticsearch/client/python-api/master/integrations.html)

### CODE REVIEW

File: docs/guide/integrations.asciidoc [hunk 1]
```diff
@@ -11,6 +11,467 @@ You can find integration options and information on this page.
...
+response = client.esql.query(
+    query="FROM employees | LIMIT 500",
+    format="csv",
+)
+df = pd.read_csv(StringIO(response.body))
+print(df)
+------------------------------------
+
+However, the dataset only contains 100 records a LIMIT command is usesd to avoid
+getting ES|QL warnings about potentially missing records. This prints the
```
Comment (marciw) [inline docs/guide/integrations.asciidoc:366]:
```suggestion
Even though the dataset contains only 100 records, a LIMIT of 500 is specified to prevent
ES|QL warnings about potentially missing records. This prints the
```

Comment (marciw) [reply docs/guide/integrations.asciidoc:366]: (or at least I *think* this is what this sentence is saying)

**Review by marciw — COMMENTED** [review]
One sentence jumped out at me so just commenting on that one for now.

I like the collapsible sections 👍

**Review by pquentin — APPROVED** [review]
Thanks, this looks good to me.

I'm happy to merge this as is, but given the amout of content is significant, what are your thoughts on using a specific page for this, as was done for OpenTelemetry?

**Comment by szabosteve** [comment]
@pquentin I agree. I moved the content to a subpage under Integrations in my latest commits.

File: docs/guide/esql-pandas.asciidoc [hunk 2]
```diff
@@ -0,0 +1,453 @@
...
+=== ES|QL and Pandas
+
+The {ref}/esql.html[Elasticsearch Query Language (ES|QL)] provides a powerful
+way to filter, transform, and analyze data stored in {es}. Designed to be easy
+to learn and use, it is a perfect fit for data scientists familiar with Pandas
+and other dataframe-based libraries as ES|QL queries produce tables with named
+columns, which is the definition of dataframes.
+
+This section shows you how to use ES|QL and Pandas together to work with
+dataframes.
```
Comment (marciw) [inline docs/guide/esql-pandas.asciidoc:11]: Two minor thoughts:
* "This page" instead of "This section" now, right?
* I'm not sure this content really does show "how to use" ES|QL and Pandas... it shows an example that the reader has to then abstract or translate to their own use cases. It's more of an illustration than a how-to. Maybe rephrase?

File: docs/guide/esql-pandas.asciidoc [hunk 3]
```diff
@@ -0,0 +1,453 @@
+[[esql-pandas]]
+=== ES|QL and Pandas
+
+The {ref}/esql.html[Elasticsearch Query Language (ES|QL)] provides a powerful
+way to filter, transform, and analyze data stored in {es}. Designed to be easy
+to learn and use, it is a perfect fit for data scientists familiar with Pandas
+and other dataframe-based libraries as ES|QL queries produce tables with named
+columns, which is the definition of dataframes.
```
Comment (marciw) [inline docs/guide/esql-pandas.asciidoc:8]:
```suggestion
and other dataframe-based libraries. ES|QL queries produce tables with named
columns, which is the definition of dataframes.
```

Comment (marciw) [reply docs/guide/esql-pandas.asciidoc:8]: (just to make it a bit more readable)

**Review by marciw — APPROVED** [review]
Suggestion you can take or leave as you see fit! :)

**Review by pquentin — APPROVED** [review]
Thanks! Again, this looks good to me and can be merged, but is it useful to add a link from integrations to the new page, as is done for OpenTelemetry?

**Review by pquentin — APPROVED** [review]

**Comment by pquentin** [comment]
Thanks for iterating @szabosteve and for the previous help @marciw! This is great.
